src/main/java/AreaCalculator.java

```diff
@@ -0,0 +1,3 @@
+public interface AreaCalculator {
+    abstract double areaValue();
```

Comment:
```suggestion
    double getArea();
```

нам не нужно дополнительно писать ключевое слово `abstract`, в интерфейсах все методы без реализации по умолчанию абстрактные. Названия методов должны быть глаголами или начинаться с глаголов